ios/FluentUI.Demo/FluentUI.Demo/Demos/ContactCollectionViewDemoController.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class ContactCollectionViewDemoController: DemoController {
```

Comment:
Doesn't need to inherit from DemoController does it? Could it just be UIViewController?

Reply:
Good catch. I agree with inheriting form `UIViewController` rather than `DemoController`. Originally I was inheriting from `DemoController` because I was using the `addRow()` method.